lib/board.rb

```diff
@@ -0,0 +1,54 @@
+class Board
+  require 'pry'
+  attr_accessor :cells, :board
+
+  def initialize
+    @board = Array.new(9, " ")
+    @cells = @board
```

Comment:
Not sure why you're using two different instance variables to store the same values.